prefect_firebolt/__init__.py

```diff
@@ -1,5 +1,6 @@
 from . import _version
 from .credentials import FireboltCredentials
+from .database import FireboltDatabase
```

Comment:
Do we want to import in __init__?

Reply:
Yes, for smaller collections all block should be top level so that we don't need to declare multiple `entry_points`. Larger collections will have an entrypoint per module. I expect this collection will be on the small side.